nodejs/awsx/examples/api/authfunction/index.js

```diff
@@ -0,0 +1,44 @@
+// Example modified from https://docs.aws.amazon.com/apigateway/latest/developerguide/apigateway-use-lambda-authorizer.html
```

Comment:
why 'js' instead of 'ts' here?

Reply:
The example was copied & then modified from an AWS example. I can change it to TS!

Reply:
Already talk to you about this: but this is getting packaged up using `pulumi.asset.FileArchive("./authfunction")`